pkg/controller/atlasproject/encryption_at_rest.go

```diff
@@ -206,14 +180,14 @@ func normalizeAwsKms(ctx *workflow.Context, projectID string, awsKms *mongodbatl
 	}
 
 	// assume that role ID is set as AWS ARN
-	resp, _, err := ctx.Client.CloudProviderAccess.ListRoles(ctx.Context, projectID)
+	resp, _, err := ctx.SdkClient.CloudProviderAccessApi.ListCloudProviderAccessRoles(ctx.Context, projectID).Execute()
```

Comment:
This call should also be wrapped within the translation layer, don't you think?

Reply:
Cloud Provider Access is a separate API which was migrated to the new SDK without adding a translation layer (in `cloud_provider_integration.go`). Adding a translation layer for this seems beyond the scope of this PR

Reply:
agreed, translation layer was skipped for https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1402

Reply:
Ok, understood. Could we make sure we have a Jira ticket for tackling this in the future?

Reply:
that's a great idea, added https://jira.mongodb.org/browse/CLOUDP-285127